Gemfile

```diff
@@ -81,13 +81,14 @@ gem 'counter_culture'
 gem 'unicorn'
 # Datetime Picker for Bootstrap3
 gem 'bootstrap3-datetimepicker-rails', '~> 4.17.47'
-gem 'momentjs-rails', '~> 2.17.1'
+gem 'momentjs-rails'
 # https://github.com/alexspeller/non-stupid-digest-assets
 gem 'non-stupid-digest-assets'
 # Upload images to Cloud
-# gem 'paperclip'
-gem 'paperclip', git: 'https://github.com/thoughtbot/paperclip.git', ref: '915c904d9a1382f53f4dcc822878ebb7c2024bb6'
-gem 'paperclip-azure', '~> 0.3.0'
+gem 'paperclip'
+# gem 'paperclip-azure'
+gem 'paperclip-azure', git: "https://github.com/kinkade/paperclip-azure", branch: "api_update_azure_storage"
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [Sider](https://sider.review) -->